code/src/index.css

```diff
@@ -11,3 +11,115 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+body {
+  background : black;
+  color: white;
+}
+
+.App {
+  display:flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.header-section p {
+  text-align: center;
+}
+
+.headphone-image{
+  width: 200px;
+}
+
+.title {
+  font-size: 70px;
+  color: greenyellow;
+  border-bottom: 2px solid indianred;
+}
+
+.title-line-one {
+  font-size: 30px;
+  font-weight: 500;
+  color:red;
+}
+
+.title-line-two {
+  font-size: 20px;
+  font-weight: 900;
+}
+
+.form-container,
+.my-form {
```

Comment:
I do now know the best practice around this but would it not be neater making these two classes into one more generic since they share attributes?